sssom/validators.py

```diff
@@ -70,6 +71,11 @@ def check_all_prefixes_in_curie_map(msdf: MappingSetDataFrame) -> None:
     :raises ValidationError: If all prefixes not in curie_map.
     """
     prefixes = get_all_prefixes(msdf)
+    prefixes_including_builtins = add_built_in_prefixes_to_prefix_map(msdf.prefix_map)
+    msdf.prefix_map = prefixes_including_builtins
+    # TODO: If prefixes_including_builtins NOT EQUAL to msdf.prefix_map
+    # 1. Raise error OR
+    # 2. Warning that staes we added built-ins.
```

Comment:
I think its fine. I anything you can add a logging.info() - not adding built-ins to the prefix map should not be considered a problem!